libs/cardano-ledger-test/cardano-ledger-test.cabal

```diff
@@ -54,7 +54,6 @@ library
     Test.Cardano.Ledger.Generic.Functions
     Test.Cardano.Ledger.Generic.GenState
     Test.Cardano.Ledger.Generic.TxGen
-    Test.Cardano.Ledger.Generic.Types
```

Comment:
Any chance this is used by others, since it's in the test library that we're exposing?

Reply:
It should probably be mentioned in the changelog at least :thinking:

Reply:
This is not used by any module in the depository. All the functions in this module are duplicated in Functions.hs,  so I think it was just an oversight it was not removed.  I will add it to the changeLog